frontend/src/__tests__/integration/pages/modelServing/ModelServingGlobal.spec.ts

```diff
@@ -41,6 +41,7 @@ test('Edit model', async ({ page }) => {
   await page
     .getByRole('textbox', { name: 'Field list AWS_SECRET_ACCESS_KEY' })
```

Comment:
in order to make the S3 model source AWS-unspecific, shouldn't you change the field name besides the displayed name?  @manaswinidas